sources/engine/Stride.VirtualReality/VRDeviceSystem.cs

```diff
@@ -91,12 +91,6 @@ private void OnEnabledChanged(object sender, EventArgs eventArgs)
                             }
                         case VRApi.WindowsMixedReality:
                         {
-#if STRIDE_GRAPHICS_API_DIRECT3D11 && STRIDE_PLATFORM_UWP
-                            if (Windows.Graphics.Holographic.HolographicSpace.IsAvailable && GraphicsDevice.Presenter is WindowsMixedRealityGraphicsPresenter)
-                            {
-                                Device = new WindowsMixedRealityHmd();
-                            }
-#endif
```

Comment:
Might want to throw a not supported here instead of silently failing ? Granted that's what it would have done before outside of UWP